repos/spack_repo/builtin/packages/py_dask/package.py

```diff
@@ -65,12 +69,16 @@ class PyDask(PythonPackage):
     depends_on("py-click@8.0:", type=("build", "run"), when="@2023.4.1:")
     depends_on("py-click@8.1:", type=("build", "run"), when="@2023.11.0:")
     depends_on("py-importlib-metadata@4.13.0:", type=("build", "run"), when="@2023.4.0:")
```

Comment:
```suggestion
    depends_on("py-importlib-metadata@4.13.0:", type=("build", "run"), when="@2023.4.0:2024.2")
```
No longer required for newer Python versions.

Reply:
Thought I added that sorry. Thanks!